app/css/ui-kit/scroll.css

```diff
@@ -0,0 +1,11 @@
+@layer base {
+    /* Makes element scrollable on the x-axis but hides the scrollbar */
+    .scroll-x-hidden {
+        @apply overflow-x-auto;
+        -ms-overflow-style: none; /* IE and Edge */
+        scrollbar-width: none; /* Firefox */
+        &::-webkit-scrollbar {
+            display: none; /* Chrome, Opera, Safari */
+        }
+    }
+}
```

Comment:
So I like this conceptually. My only concern is whether people will know there's extra tabs they **can** scroll to if there's no scroll bar? Do we need to make that more obvious somehow?